docs/install/scala_setup.md

```diff
@@ -0,0 +1,109 @@
+# Setup the MXNet Package for Scala
+
+The following instructions are provided for macOS and Ubuntu. Windows is not yet available.
+
+**Note:** If you use IntelliJ or a similar IDE, you may want to follow the [MXNet-Scala on IntelliJ tutorial](../tutorials/scala/mxnet_scala_on_intellij.html) instead of these instructions.
+
+<hr>
+
+## Maven
+
+### Setup Instructions
+
+**Step 1.** Install dependencies:
+
+**macOS Steps**
+
+```bash
+brew update
+brew tap caskroom/versions
+brew cask install java8
+brew install opencv
+brew install maven
+```
+
+**Ubuntu Steps**
+
+These scripts will install Maven and its dependencies.
+
+```bash
+wget https://raw.githubusercontent.com/apache/incubator-mxnet/master/ci/docker/install/ubuntu_core.sh
+wget https://raw.githubusercontent.com/apache/incubator-mxnet/master/ci/docker/install/ubuntu_scala.sh
+sudo ./ubuntu_core.sh
+sudo ./ubuntu_scala.sh
+```
+
```

Comment:
Ubuntu GPU cannot be built using the same script, users are required to build on their own for the CUDA driver. Some recommended step can be seen in [here](https://medium.com/@zhanwenchen/install-cuda-and-cudnn-for-tensorflow-gpu-on-ubuntu-79306e4ac04e).

Reply:
@lanking520 the link provides CUDA installation instructions 
http://mxnet.incubator.apache.org/install/ubuntu_setup.html#installing-mxnet-on-ubuntu